src/js/components/ChatBody/ChatBody.js

```diff
@@ -0,0 +1,60 @@
+import PropTypes from 'prop-types';
+import React, { useEffect, useRef } from 'react';
+import '../../../scss/components/ChatBody/ChatBody.scss';
+import ChatLogControl from '../ChatLogControl/ChatLogControl';
+import { avatarClassNames } from '../GroupNavDMButton/GroupNavDMButton';
+
+const ChatBody = ({ title, avatarSrc, backgroundColor }) => {
+  const refChatBody = useRef(null);
+
+  /**
+   * Scroll to the bottom of the chat body
+   */
+  useEffect(() => {
+    const { current } = refChatBody;
+    current.scrollTo(0, current.scrollHeight);
+  });
+
+  /**
+   * Resolves classes for the avatar
+   * @returns {string}
+   */
+  const avatar = () => {
+    const { baseAvatar, noAvatar, availableAvatar } = avatarClassNames;
+
+    if (!avatarSrc) {
+      return [baseAvatar, noAvatar, backgroundColor].join(' ');
+    }
+    return [baseAvatar, availableAvatar].join(' ');
+  };
+
+  return (
+    <div className="chat-body" ref={refChatBody}>
+      <div className="chat-log-item chat-log-item-spacer" />
+      <div className="chat-log-item chat-log-item-head">
+        <div className={avatar()} />
+        <div className="log-head-title">{title}</div>
+        <div className="log-head-text">
+          This is the beginning of your direct message history with
+          <strong>{` @${title}`}</strong>
```

Comment:
Does the leading ` ` (space) need to be `strong`?

Reply:
Yea, eslint won't allow a space after 'with'